docs/modules/rust-guide/pages/rust-quickstart.adoc

```diff
@@ -263,9 +251,10 @@ To see details about the Candid interface description language syntax, see the l
 
 To add a Candid file for this tutorial:
 
-. Check that you are still in the project `+src/rust_hello+` source code directory, if needed.
-. Create a new file named `+rust_hello.did+`.
-. Open the file in a text editor, then copy and paste the following `+service+` definition for the `+print+` function:
+. Check that you are still in the root directory for your project, if needed.
+. Create a new file in the `+src/rust_hello/+` directory named `+rust_hello.did+`.
+. Open the `+src/rust_hello/src/rust_hello.did+`file in a text editor.
```

Comment:
```suggestion
. Open the `+src/rust_hello/src/rust_hello.did+` file in a text editor.
```

I think adding this space will make the formatting work right, otherwise it renders a bit odd for me (on github at least) https://github.com/dfinity/cdk-rs/blob/3c5e80ae00bd5eab124e99203966bf2b3cfee8f8/docs/modules/rust-guide/pages/rust-quickstart.adoc#add-an-interface-description-file